pkg/v1/tkg/client/kubevip_update_helper.go

```diff
@@ -51,9 +51,29 @@ func ModifyKubeVipTimeOutAndSerialize(currentKubeVipPod *corev1.Pod, newLeaseDur
 		}
 		envVars = append(envVars, envVar)
 	}
-
+	envVar := corev1.EnvVar{Name: "cp_enable", Value: "true"}
+	envVars = append(envVars, envVar)
 	currentKubeVipPod.Spec.Containers[0].Env = envVars
 
+	// replace arg "start" with "manager"
+	for index, arg := range currentKubeVipPod.Spec.Containers[0].Args {
+		if arg == "start" {
+			currentKubeVipPod.Spec.Containers[0].Args[index] = "manager"
+		}
+	}
+
+	// Replace capability SYS_TIME with NET_RAW
+	for index, capability := range currentKubeVipPod.Spec.Containers[0].SecurityContext.Capabilities.Add {
+		if capability == "SYS_TIME" {
+			currentKubeVipPod.Spec.Containers[0].SecurityContext.Capabilities.Add[index] = "NET_RAW"
+		}
+	}
+	currentKubeVipPod.Spec.HostAliases = []corev1.HostAlias{
+		{
+			IP:        "127.0.0.1",
+			Hostnames: []string{"kubernetes"},
+		},
+	}
```

Comment:
nit: IMO all the hardcoded strings should be made constants, and ought to be referenced if they're already constants. Thoughts?